src/app/feed/feed-not-found/feed-not-found.component.html

```diff
@@ -11,5 +11,10 @@
 			<li>Try different keywords, accounts or hashtags.</li>
 			<li>Try more general keywords, accounts or hashtags.</li>
 		</ul>
+
+		<p>
+			If you encountered an error of the system, please submit a bug report in the
```

Comment:
How about something this?

`If the problem persists, please submit a bug report to the loklak issue tracker.`